api/apps/geoprocessing/test/index.html

```diff
@@ -42,58 +42,67 @@
         style: {
           version: 8,
           sources: {
-            'postgis-tiles0': {
-              type: 'vector',
-              maxzoom: 12,
-              tiles: [
-                'http://localhost:3040/api/v1/administrative-areas/0/preview/tiles/{z}/{x}/{y}.mvt',
-              ],
-            },
-            'postgis-tiles1': {
-              type: 'vector',
-              maxzoom: 12,
-              tiles: [
-                'http://localhost:3040/api/v1/administrative-areas/1/preview/tiles/{z}/{x}/{y}.mvt?guid=BWA',
-              ],
-            },
-            'postgis-tiles2': {
-              type: 'vector',
-              maxzoom: 12,
-              tiles: [
-                'http://localhost:3040/api/v1/administrative-areas/2/preview/tiles/{z}/{x}/{y}.mvt?guid=BWA.12_1&bbox=[21.14044189453125,-19.202241064923044,22.980651855468746,-18.22935133838667]',
-              ],
-            },
-            'postgis-tileswdpa': {
-              type: 'vector',
-              minzoom: 1,
-              maxzoom: 12,
-              tiles: [
-                'http://localhost:3040/api/v1/protected-areas/preview/tiles/{z}/{x}/{y}.mvt',
-              ],
-            },
+            // 'postgis-tiles0': {
```

Comment:
there's a lot of commented-out source in this file - can we remove it?
if it's not used only temporarily, maybe these sources could be moved temporarily to a `skippedSources` prop or something like that.

but it's a test file, so feel free to leave it as is, of course 😅

Reply:
yes is our sandbox for map testing, so ideally each layer and style represents one of our designed tiles endpoints, and in this specific I was avoiding noise.